docs/tutorial.md

```diff
@@ -4,16 +4,20 @@ This Python client communicates with the Avatar platform.
 
 For more information about the Avatar method and process, check out our main docs at https://docs.octopize.io
 
+## Prerequisites
+
+Avatar-python is intended to be run with Python 3.9.
+
```

Comment:
Make sure that this does not mess with the `make update-python-tutorial` in the `public-docs` repo.